deploy/helm/templates/rbac/clusterrole.yaml

```diff
@@ -19,7 +19,16 @@ rules:
 {{- $doInclude := false }}
 {{- range $i, $line := .Files.Lines "config/rbac/role.yaml" }}
   {{- if eq $doInclude true }}
-{{- $line | nindent 2 }}
+    {{- $line | nindent 2 }}
   {{- end }}
   {{- if eq $line "rules:" }}{{- $doInclude = true }}{{- end }}
 {{- end }}
+{{- if eq .Values.rbac.enabled  true }}
+  {{- $doInclude := false }}
```

Comment:
have template contents instead of load from a file, do aware that `config/rbac/role.yaml` is generated contents, and `config/rbac/rbac_manager_role.yaml` is not.